api/types/common/common.go

```diff
@@ -16,22 +16,43 @@ limitations under the License.
 
 package common
 
-import "regexp"
+// IsValidLabelKey checks if the supplied string matches the
+// label key regexp.
```

Comment:
```suggestion
// IsValidLabelKey checks if the supplied string is a valid label key. 
```

The fact that a regexp is used is an internal implementation detail that doesn't need to be mentioned in the godoc. 

In fact, we're not even using a regexp at all anymore!